_scripts/linting.js

```diff
@@ -54,6 +55,25 @@ dirs.forEach(dir => {
             exit(1);
         }
     });
+
+    const trackerRadarFiles = glob.sync("/**/tracker_radar*.json", { root: path.resolve(root, featureFolderName) });;
+
+    trackerRadarFiles.forEach(testFile => {
+        const testBasename = path.basename(testFile);
+
+        console.log(` - validating ${testBasename}`);
+
+        const trackerRadarObject = JSON.parse(fs.readFileSync(testFile));
+
+        const errors = checkTrackerRadar(trackerRadarObject);
+
+        if (errors.length) {
+            console.error(` ❌ ${testFile} is not a valid Tracker Radar blocklist:`);
+            console.error(errors.map(item => `  - ${item.location}: ${item.message}`).join('\n'));
+            exit(1);
+        }
+    })
```

Comment:
the output looks like this:

![tracker-radar-erros](https://user-images.githubusercontent.com/985504/145412055-427eabba-ab4b-4e50-a514-df1c3f27989a.jpg)

Reply:
Given this test was passing in the extension and iOS, seems like we have an ambiguity in implementation.  Given the point of these tests, shouldn't we have both versions (incorrect and correct) and validate the correct behavior across the products?